### PR TITLE
bump upstream dependencies

### PR DESCRIPTION
**Pull request by myronkscott**

### CODE REVIEW

File: clustered/server/build.gradle [hunk 1]
```diff
@@ -35,4 +35,5 @@ dependencies {
   providedImplementation "org.terracotta:runnel:$terracottaPlatformVersion"
 
   testImplementation project(':clustered:test-utils')
+  testImplementation ("org.terracotta:passthrough-server:$terracottaPassthroughTestingVersion")
```
Comment (chrisdennis) [inline clustered/server/build.gradle]: Why does this need to be added?

File: clustered/clustered-dist/build.gradle [hunk 2]
```diff
@@ -79,7 +79,7 @@ jar {
     'Bundle-SymbolicName': 'org.ehcache.clustered',
     'Export-Package': '!com.tc.*, !com.terracotta.*, !org.terracotta.*, !org.ehcache.*.internal.*, !sun.misc, ' +
       'org.ehcache.clustered.client.*, org.ehcache.clustered.common.*',
-    'Import-Package': '!sun.misc.*, org.ehcache.xml.*;resolution:=optional, *'
+    'Import-Package': '!sun.misc.*, org.ehcache.xml.*;resolution:=optional, jdk.jfr.*;resolution:=optional, *'
```
Comment (chrisdennis) [inline clustered/clustered-dist/build.gradle]: This makes sense. Have you confirmed however that this stuff does work correctly if the bundle is available?

File: build.gradle [hunk 3]
```diff
@@ -215,6 +218,7 @@ subprojects {
     }
   }
 
+
```
Comment (chrisdennis) [inline build.gradle]: Random whitespace change.

File: build.gradle [hunk 4]
```diff
@@ -149,6 +149,9 @@ subprojects {
     if (parent.isCloudbees) {
       systemProperty 'disable.concurrent.tests', 'true'
     }
+    jacoco {
+        excludes = ['org.terracotta.tripwire.*']
```
[the remark this answers is not in the document]
Comment (myronkscott) [reply build.gradle]: The JFR custom event classes are already instrumented with some magic from that package.  Jacoco doesn't like that and this error is generated during tests.  Caused by: java.lang.IllegalStateException: Cannot process instrumented class XXX. Please supply original non-instrumented classes

File: clustered/clustered-dist/build.gradle (hunk 2 above)
Comment (myronkscott) [reply clustered/clustered-dist/build.gradle]: I've confirmed everything works if everything is present and a tripwire appender is added to logback.

File: clustered/server/build.gradle (hunk 1 above)
Comment (myronkscott) [reply clustered/server/build.gradle]: Offheap is now instrumented with tripwire which is provided by the server.  There are some unit tests that instantiate OffheapProvider without a server so passthrough is added to provide the required libs.  We could just add tripwire but that is one more dependency to manage.

Comment (chrisdennis) [reply clustered/server/build.gradle]: Okay.

File: build.gradle (hunk 4 above)
[the remark this answers is not in the document]
Comment (chrisdennis) [reply build.gradle]: Okay, that makes sense. Could we switch to `excludes += 'org.terracotta.tripwire.*'`?

[the remark this answers is not in the document]
Comment (myronkscott) [reply build.gradle]: ok

**Pull request merged**